### PR TITLE
Support for private packages

### PR DESCRIPTION
**Pull request by nicohaemhouts**
Replacing `/` with `-` in the package name as discussed in issue https://github.com/SamVerschueren/alfred-link/issues/15

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ exports.link = opts => {
 			const filePath = result.path;
 
 			const src = path.dirname(filePath);
-			const dest = path.join(workflowDir, pkg.name);
+			const dest = path.join(workflowDir, pkg.name.replace('/', '-'));
```
Comment (SamVerschueren) [inline index.js:43]: Is it possible to have multiple slashes? Probably not, just wondering. Otherwise we would have to do `.replace(/\//g, '-')`.

Comment (sindresorhus) [reply index.js:43]: No, only one slash is possible.

**Comment by sindresorhus** [comment]
Thanks @nicohaemhouts :)

**Comment by nicohaemhouts** [comment]
Hey @sindresorhus , thanks for the merge :-) When do you reckon you'd be able to publish a patch to npm?

**Comment by SamVerschueren** [comment]
Will try to do it today. 

@sindresorhus if you can do it, go ahead. Not sure if I added you to npm as well.

**Comment by SamVerschueren** [comment]
Thanks for the PR by the way :) :tada:

**Comment by SamVerschueren** [comment]
Did a `0.2.1` release just now.

**Comment by nicohaemhouts** [comment]
Awesome, thanks 👍

**Pull request merged**